kstatemachine/src/commonMain/kotlin/ru/nsk/kstatemachine/TransitionBuilder.kt

```diff
@@ -3,21 +3,21 @@ package ru.nsk.kstatemachine
 import ru.nsk.kstatemachine.TransitionDirectionProducerPolicy.*
 
 @StateMachineDslMarker
-abstract class TransitionBuilder<E : Event>(protected val name: String?, protected val sourceState: IState) {
+abstract class TransitionBuilder<E : Event>(protected val name: String?, protected val metaInfo: TransitionMetaInfo?, protected val sourceState: IState) {
```

Comment:
is should not be protected as naibours?

Reply:
neither of parameters is used in child builder directly. should all of them stay protected or can be just private?